src/Export/Classes/GGPKData.lua

```diff
@@ -67,48 +63,44 @@ function GGPKClass:ExtractFilesWithBun(fileListStr, useRegex)
 	os.execute(cmd)
 end
 
+-- Use manifest files to avoid command line limit and reduce cmd calls
+function GGPKClass:ExtractFilesWithBunFromTable(fileTable, useRegex)
+	local useRegex = useRegex or false
+	local manifest = self.oozPath .. "extract_list.txt"
```

Comment:
Where does this file come from?

Reply:
This is just the path to the file. The next line creates it as a temp file before removing it at the end

Reply:
Oh sure, I misunderstood what you were trying to do here, makes sense.